web/sites/default/config/views.view.alerts_page.yml

```diff
@@ -0,0 +1,204 @@
+uuid: 2cd8ec86-f937-47d6-baf0-9d382888b24a
```

Comment:
Why did you move this from the `alert` view into a completely separate view? We should be trying to consolidate our views whenever possible and I don't see any reason why this needs to be a separate view.

Reply:
I  used a different view in order to apply a different display mode to the alerts that appear on the alerts page versus globally on the site. This allows for alerts to appear on the alerts page without the dismissible button.